src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +27,14 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(x -> x.length() > 12)
+                .distinct()
+                .sorted((x, y) -> y.length() - x.length())
```

Comment:
```suggestion
        words.stream()
                .filter(x -> x.length() > 12)
                .distinct()
                .sorted(String::compareTo)
```

String은 Comparable을 구현하고 있어 비교가 가능합니다.